### ISSUE TITLE
SA1013 seems to be unaware of object initializers

### ISSUE THREAD

**Issue by RalfKoban** [DotNetAnalyzers/StyleCopAnalyzers#1078]
Hi,

I'm using StyleCopAnalyzers 1.0.0-beta2 (but also on beta1).
It seems that the checks for rule SA1013 is not aware of an object initializers, as in following snippet:

```csharp
var obj = new MyObject { Something = 42 }.As<ISomething>();
```

It reports that there is a space missing between the `}` and `.` but actually there should be no space in between (which then would violate SA1019).

BR,
Ralf

**Comment by pdelvo** [comment]
Im going to take a look at it

**Comment by RalfKoban** [comment]
Works. :)

BTW: Cool tool. :+1:
